content/400-reference/200-api-reference/050-prisma-client-reference.mdx

```diff
@@ -4247,6 +4247,371 @@ const posts = await prisma.post.findMany({
 })
 ```
 
+## Composite type methods
```

Comment:
Should we call this a method, I'm lending more towards calling it "operations". wdyt?

Reply:
Ah, I was going for consistency with scalar field methods and filters elsewhere in the reference. Will leave for now as may have to be changed in a number of places, but happy to update later.